Network/Connection.cs

```diff
@@ -600,6 +605,30 @@ internal void Send(Packet packet, object instance, bool ignoreWriteLock)
             dataAvailableEvent.Set();
         }
 
+        /// <summary>
+        /// Sends the given <see cref="RawData"/> packet to the network.
+        /// </summary>
+        /// <param name="rawData">The <see cref="RawData"/> packet to send to the network.</param>
+        public void SendRawData(RawData rawData) => Send(rawData);
+
+        /// <summary>
+        /// Sends the given raw, serialised primitive to the network.
+        /// </summary>
+        /// <param name="key">
+        /// The <see cref="string"/> key which identifies the raw data <see cref="PacketReceivedHandler{P}"/> to use for the data.
+        /// </param>
+        /// <param name="data">The serialised raw primitive, as a <see cref="byte"/> array.</param>
+        public void SendRawData(string key, byte[] data)
```

Comment:
Why is this method in the Connection.cs?

Reply:
makes more sense to put the SendXXX methods together? They all use the same underlying methods so they should (in my mind) be grouped together.

Reply:
same